plugins/DB.cpp

```diff
@@ -46,12 +47,29 @@ bool BedrockPlugin_DB::peekCommand(SQLite& db, BedrockCommand& command) {
         //         read-write command is mis-classified as read-only an executed in
         //         the peek, but even then we'll detect it after the fact and shut
         //         the node down.
-        const string& query = request["query"] + ";";
+        const string& query = request["query"];
         const string& upperQuery = SToUpper(STrim(query));
+        const string& nowhere = request["nowhere"];
+        const bool shouldRequireWhere = !nowhere.size() || SToUpper(nowhere) == "false";
```

Comment:
`nowhere.empty()`?

And:

`SToUpper(nowhere) == "false"` <-- do you mean `SToLower()`?